packages/web/utils/questChains.ts

```diff
@@ -48,87 +61,138 @@ export const Time = {
   LONG: 'Long',
 };
 
-export type QuestChainType = Values<typeof QuestChains>;
+export type QuestChainType =
+  | Values<typeof QuestChainsRoles>
+  | Values<typeof QuestChainsGreatHouses>;
 
 type QuestChainInfo = {
   chainId: string;
   address: string;
   title: string;
   description: string;
-  icon: string;
-  difficulty: string;
-  time: string;
+  image: string;
+  difficulty?: string;
+  time?: string;
 };
 
-export const QuestChainDetails: Record<QuestChainType, QuestChainInfo> = {
+export const QuestChainRolesDetails: Record<QuestChainType, QuestChainInfo> = {
   // Engaged Octo's path: https://questchains.xyz/chain/0x89/0xea512722CC047bfDbe90a1a2750491e89a818CA7
-  [QuestChains.ENGAGED_OCTOS_PATH]: {
```

Comment:
Seems like all this stuff should be in a table in the backend rather than hardcoded. Bit of a lift, though

Reply:
Then people like peth could potentially add new ones?

Reply:
this would be really nice, yeah - but this should be a whole new, separate feature imo